test/studies/prk/stencil/stencil-pretty.chpl

```diff
@@ -198,6 +201,7 @@ proc main() {
     if useStencilDist then {
       if debug then diagnostics('output.updateFluff()');
       output.updateFluff();
```

Comment:
Thinking about this a little more, I wonder if `output` could just be a BlockDist instead of a StencilDist.